reducers/views/channel.js

```diff
@@ -46,7 +46,11 @@ function lastChannelViewTime(state = {}, action) {
         }
         return state;
     }
-
+    case ActionTypes.UPDATE_CHANNEL_LAST_VIEWED_AT: {
```

Comment:
`lastChannelViewTime` is also used for ensuring the channel is in sync and up to date. If we're changing that here, could that cause us to miss messages possibly? Perhaps the new messages line should be given its own field.